datasets/oscar/README.md

```diff
@@ -1,4 +1,5902 @@
+}
+```
+
+#### unshuffled_original_xal
+
+- **Size of downloaded dataset files:** 0.03 MB
+- **Size of the generated dataset:** 0.11 MB
+- **Total amount of disk used:** 0.14 MB
+
+An example of 'train' looks as follows.
+```
+This example was too long and was cropped:
+
+{
+    "id": 0,
+    "text": "\"Арнгудин Орн гисн Европд бәәдг һазр. 2007 җилин тooһaр эн орн нутгт 3,600,523 әмтн бәәдг билә. Арнгудин Орнин хотл балһсна нерн..."
+}
+```
+
+#### unshuffled_original_xmf
+
+- **Size of downloaded dataset files:** 1.00 MB
+- **Size of the generated dataset:** 5.84 MB
+- **Total amount of disk used:** 6.84 MB
+
+An example of 'train' looks as follows.
+```
+This example was too long and was cropped:
+
+{
+    "id": 0,
+    "text": "\"მოჩამილი ტექსტი წჷმორინელი რე Creative Commons Attribution-ShareAlike ლიცენზიათ; შილებე გეძინელი პირობეფიშ არსებუა. კილიშკილიშა..."
+}
+```
+
+#### unshuffled_original_yi
+
+- **Size of downloaded dataset files:** 31.79 MB
+- **Size of the generated dataset:** 140.76 MB
+- **Total amount of disk used:** 172.56 MB
+
+An example of 'train' looks as follows.
+```
+This example was too long and was cropped:
+
+{
+    "id": 0,
+    "text": "\"ממשותדיק - חבֿרה, איך אַרבעט איצט אױף אַ זשורנאַל. טאָמער איר האָט עפּעס צוצוגעבן זאָלט איר שיקן מיר אַן אָנזאָג. ס'װעט הײסן \\\"..."
+}
+```
+
+#### unshuffled_original_yo
+
+- **Size of downloaded dataset files:** 0.01 MB
+- **Size of the generated dataset:** 0.06 MB
+- **Total amount of disk used:** 0.06 MB
+
+An example of 'train' looks as follows.
+```
+This example was too long and was cropped:
+
+{
+    "id": 0,
+    "text": "\"Copyright © 2018 BBC. BBC kò mọ̀ nípa àwọn ohun tí ó wà ní àwọn ojú òpó tí ó wà ní ìta. Ọwọ́ tí a fi mú ìbáṣepọ̀ ti ìta.\"..."
+}
+```
+
+#### unshuffled_original_yue
+
+- **Size of downloaded dataset files:** 0.00 MB
+- **Size of the generated dataset:** 0.00 MB
+- **Total amount of disk used:** 0.00 MB
+
+An example of 'train' looks as follows.
+```
+This example was too long and was cropped:
+
+{
+    "id": 1,
+    "text": "\"我 灌 我 灌 我 灌 灌 灌 我 灌 我 灌 我 灌 灌 灌 我 灌 我 灌 我 灌 灌 灌 我 灌 我 灌 我 灌 灌 灌 我 灌 我 灌 我 灌 灌 灌 我 灌 我 灌 我 灌 灌 灌 你還不爆 我累了 投降輸一半可以嗎\"..."
+}
+```
+
+#### unshuffled_original_zh
+
+- **Size of downloaded dataset files:** 196460.72 MB
+- **Size of the generated dataset:** 520331.90 MB
+- **Total amount of disk used:** 716792.61 MB
+
+An example of 'train' looks as follows.
+```
+This example was too long and was cropped:
+
+{
+    "id": 1,
+    "text": "\"中国铝灰网 中国有色金属矿产网 中国黄莲网 中国水轮发电机网 中国抽油泵网 中国数控雕刻机网 中国不锈钢抛光网 中国磨具加工网 中国压铸铝网 中国耐水腻子网 中国手机摄像头网 中国粗粮网 中国车门锁网 中国钛粉网 中国轮圈网\\n天天中奖彩票图 天天中彩票..."
+}
+```
+
+</details>
+
+### [Data Fields](#data-fields)
+
+The data fields are the same among all configs.
+
+- `id`: a `int64` feature.
+- `text`: a `string` feature.
+
+### [Data Splits Sample Size](#data-splits-sample-size)
+
```

Comment:
```suggestion


<details>
  <summary>Click to expand the number of samples per configuration</summary>

```
Maybe we can collapse this as well since there's already the table of the `Languages` section